Sources/Segment/ObjC/ObjCConfiguration.swift

```diff
@@ -13,6 +13,9 @@ import Foundation
 public class ObjCConfiguration: NSObject {
     internal var configuration: Configuration
     
+    /// Sets a reference to your application.  This can be useful in instances
+    /// where referring back to your application is necessary, such as within plugins
+    /// or async code.  The default value is `nil`.
     @objc
     public var application: Any? {
```

Comment:
Any chance this should be a WEAK reference?

Reply:
Yeah, good call.

Reply:
actually, not necessary.  it's a pass through and is already held weak.